api/db/apds.js

```diff
@@ -9,13 +9,10 @@ const createAPD = async (apd, { APD = mongoose.model('APD') } = {}) => {
   return newApd._id.toString(); // eslint-disable-line no-underscore-dangle
 };
 
-const deleteAPD = async (id, { APD = mongoose.model('APD') } = {}) => {
-  const result = await APD.deleteOne({ _id: id });
-  return result;
-};
-
-const deleteAPDByID = async (id, { db = knex } = {}) => {
-  await db('apds').where('id', id).update({ status: 'archived' });
+const deleteAPDByID = async (id, { APD = mongoose.model('APD') } = {}) => {
+  const apd = await APD.findById(id).exec();
```

Comment:
Why not use a `.findByIdAndUpdate()` here?

Reply:
good call, I'll updated it

Reply:
I updated it to be more streamlined, but findByIdAndUpdate is deprecated so I went with updateOne